templates/default/scripts/test_integration.sh

```
@@ -19,14 +19,14 @@ failed_message() {
   exit 1
 }
 
-export $(enclave print-env --chain hardhat)
+export $(enclave print-env --chain localhost)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)